src/app/Http/Controllers/Auth/LoginController.php

```diff
@@ -0,0 +1,39 @@
+<?php
+
+namespace App\Http\Controllers\Auth;
+
+use App\Http\Controllers\Controller;
+use App\Http\Requests\AuthRequest;
+use Illuminate\Support\Facades\Auth;
+
+/**
+ * LoginController
+ */
+class LoginController extends Controller
+{
+    /**
+     * ログイン画面表示
+     */ 
+    public function getLoginPage()
+    {
+        return view('auth.login');
+    }
+
+    /**
+     * ログイン処理
+     */
+    public function authenticateUser(AuthRequest $request)
+    {
+        $cledentials = $request->validated();
+        
+        if (Auth::attempt($cledentials)) {
+            $request->session()->regenerate();
+
+            return redirect()->intended(route('todos.index'));
+        }
+
+        return redirect()->route('auth.login')->withErrors([
+            'authError' => 'メールアドレスまたはパスワードが一致しません。',
+        ]);
```

Comment:
ユーザー登録のPRでコメントし損ねたけど、「メールアドレスまたはパスワードが一致しません。」という文字は、本来はview側で担保すべきかな！
（修正するかは任せます笑）

個人的には下記のようにbooleanのフラグだけviewへ渡して、そのbooleanをviewで判定してエラーテキストを表示する方がすき笑


```suggestion
        return redirect()->route('auth.login')->with('isLoginError', true);
```